app/controllers/api/tomatoes_controller.rb

```diff
@@ -39,6 +42,14 @@ def destroy
 
     private
 
+    def from
+      @from ||= Time.zone.parse(params[:from].to_s)
```

Comment:
Yes, that's how. `#parse` should work on any string, it returns `nil` in case of parsing failure.